### ISSUE TITLE
Cannot move cursor to the last slide, at overview keyboard navigation.

### ISSUE THREAD

**Issue by VergeDX** [slidevjs/slidev#1016]
**Describe the bug**

"Slides overview" (https://sli.dev/guide/navigation.html#slides-overview)  
allow using keyboard navigation to select each slide's thumbnail,  
but I noticed it cannot select the last slide.

**To Reproduce**

1. Go to the start slides.
2. Click "Slides Overview" button of "Navigation Bar".
3. Use four arrow keys, try to select the last slide.

**Desktop (please complete the following information):**

 - OS: NixOS 22.11
 - Browser: Chromium 113.0.5672.126
 - Slidev version: 0.42.3
